### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by Starbuck5**
README:
Drop py2 badge, update commits to base against 2.1, remove some obsolete badges
(I kept LGTM badges since we might get that working again in the medium term)

Cursor doc:
Added some more info about system cursor cross platform support, to satisfy #2832 

Chimp example:
In another PR, I messed up the fist image, so I fixed that. I can't believe I didn't notice it breaking last time.
I also changed it so the hand is in front of the chimp, not the other way around.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -155,20 +154,16 @@ See docs/licenses for licenses of dependencies.
...
 .. |PyPiVersion| image:: https://img.shields.io/pypi/v/pygame.svg?v=1
    :target: https://pypi.python.org/pypi/pygame
 
 .. |PyPiLicense| image:: https://img.shields.io/pypi/l/pygame.svg?v=1
    :target: https://pypi.python.org/pypi/pygame
 
-.. |Python2| image:: https://img.shields.io/badge/python-2-blue.svg?v=1
 .. |Python3| image:: https://img.shields.io/badge/python-3-blue.svg?v=1
 
 .. |GithubCommits| image:: https://img.shields.io/github/commits-since/pygame/pygame/2.0.2.svg
```
Comment (ankith26) [inline README.rst:165]: This link should be updated to https://img.shields.io/github/commits-since/pygame/pygame/2.1.0.svg

**Review by ankith26 — APPROVED** [review]
👍 🎈

**Comment by PokeyManatee4** [comment]
Good 🔰

**Review by zoldalma999 — APPROVED** [review]
Looks good to me!

**Pull request merged**